src/util/grave.dsc

```diff
@@ -1,14 +1,14 @@
 mortal_create_grave:
   type: task
-  definitions: location
+  definitions: location|player
   script:
-  - if !<player.inventory.is_empty>:
+  - if !<[player].inventory.is_empty>:
     # Place skull with random rotation
     - modifyblock <[location]> player_head[direction=<material[skeleton_skull].valid_directions.random>]
-    - adjust <[location]> skull_skin:<player.skull_skin>
+    - adjust <[location]> skull_skin:<[player].skull_skin>
     # Generate and set grave inventory
-    - define inv "<inventory[generic[size=54;title=<player.name>'s Grave]]>"
-    - inventory copy d:<[inv]> o:<player.inventory>
+    - define inv "<inventory[generic[size=54;title=<[player].if_null[<player>].name>'s Grave]]>"
+    - inventory copy d:<[inv]> o:<[player].inventory>
     - flag <[location]> mortal.grave:<[inv]>
     # Clear player's inventory
-    - inventory clear
\ No newline at end of file
+    - inventory clear d:<[player].inventory>
```

Comment:
why

Reply:
how tf do i deal with the target player?

Reply:
i see

Reply:
you *could* also define `__player` (which `<player>` internally uses)